eslint-config-base/index.js

```diff
@@ -32,16 +32,14 @@ module.exports = {
     "eslint:recommended",
     "plugin:jest/recommended",
     "plugin:jest/style",
-    "plugin:@typescript-eslint/recommended",
     "plugin:prettier/recommended",
-    "prettier/@typescript-eslint"
   ],
 
   // Set up es6 and typescript linting, and add lint rules for jest
```

Comment:
Is this comment out-of-date now...? Not sure it was adding anything in the first place even...